interface/usergroup/facility_user_admin.php

```diff
@@ -140,56 +160,62 @@
             <div class="col-12">
                 <div class="page-title">
                     <h3><?php echo xlt('Edit Facility Specific User Information'); ?></h3>
+                    &nbsp;
                 </div>
             </div>
         </div>
         <div class="row">
             <form name='form_facility_user' id='form_facility_user' method='post' action="facility_user.php">
                 <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
                 <input type=hidden name=mode value="facility_user_id">
-                <input type=hidden name=user_id value="<?php echo attr($_GET["user_id"]);?>">
-                <input type=hidden name=fac_id value="<?php echo attr($_GET["fac_id"]);?>">
+                <input type=hidden name=user_id value="<?php echo attr($_GET["user_id"]); ?>">
+                <input type=hidden name=fac_id value="<?php echo attr($_GET["fac_id"]); ?>">
                 <?php $iter = sqlQuery("select * from facility_user_ids where id=?", array($my_id)); ?>
 
                 <table border=0 cellpadding=0 cellspacing=0>
                     <tr>
                         <td>
-                            <?php echo xlt('User'); ?>:
+                            <p><?php echo xlt('User'); ?>:</p>
                         </td>
                         <td>
-                            <?php echo text($user_info['username']); ?>
+                            <p> <?php echo text($user_info['username']); ?> <p>
```

Comment:
Please add the bootstrap table style, It may look like this
```html
<table class="table table-borderless ">
```

Also, I guess the `td` content didn't need to add the `p` tag.
cc @tywrenn

Reply:
Thanks @stu01509 for the review. I think I have fixed some of them :)